writ-docker

```diff
@@ -0,0 +1,72 @@
+#!/bin/env bash
+
+DOCKER=$(command -v docker)
+if [ -z "${DOCKER}" ] ; then
+    echo "Docker was not found in your PATH.  Aborting."
+    exit 1
+fi
+
+usage() {
```

Comment:
I'm not sure why do you rewrite a separate script to validate input instead of reusing `validate_input.py`. Is this for something Docker-specific?